adapters/handlers/rest/configure_api.go

```diff
@@ -179,6 +179,12 @@ func MakeAppState(ctx context.Context, options *swag.CommandLineOptionsGroup) *s
 		}
 
 		sentry.ConfigureScope(func(scope *sentry.Scope) {
+			// Set cluster ID and cluster owner using sentry user feature to distinguish multiple clusters in the UI
+			scope.SetUser(sentry.User{
+				ID:       appState.ServerConfig.Config.Sentry.ClusterId,
+				Username: appState.ServerConfig.Config.Sentry.ClusterOwner,
+			})
```

Comment:
**suggestion:** shall we auto set incase of empty to OS

Reply:
I don't think there would be any good "autoset" value we could set it to. In the UI it would default to "unassigned" which I think makes sense. Do you have an idea of a good default value ?

Reply:
I don't think so, the reasoning being that this would "collide" values in sentry if we were to take "hostname" for example and assign it all to a given node, but we'd have to ability to narrow that down further. I think in that case we should leave it empty and "unassigned" on the sentry UI

Reply:
I see your point, it could require an extra work but can't we use k8s client to detect. feel free to ignore if it's too much work.